docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -1,79 +1,96 @@
 ---
 sidebar_navigation:
-  title: Nextcloud integration
+  title: Nextcloud integration setup
   priority: 600
-description: Nextcloud and OpenProject integration
+description: Nextcloud integration setup
 keywords: integrations, apps, Nextcloud
 ---
 
-# OpenProject and Nextcloud integration 
+# OpenProject and Nextcloud integration setup 
 
-The OpenProject and Nextcloud integration will improve the productivity of their enterprise users. It combines the strengths of the market leading content collaboration platform Nextcloud and the leading open source project management software OpenProject.
+The integration between OpenProject and Nextcloud has the potential to improve the productivity of all users. It combines the strengths of Nextcloud, the market-leading collaboration platform, and OpenProject, the leading open source project management software.
 
-The integration is available starting with Nextcloud 20. It enables users to keep an eye on ongoing project activities directly in their Nextcloud instance.
+The integration enables users to keep an eye on ongoing project activities directly in their Nextcloud instance and link their Nextcloud files to OpenProject work packages. To learn more about how to use the integration, please refer to the [Nextcloud user guide](../../../user-guide/nextcloud-integration/)
```

Comment:
If  with "keep an eye on the ongoing project activities" you mean the dashboard, then I would deprioritize that aspect and focus more on "Where are the files that I need to be able to complete a task?" and "Where do I put the final files after completing a task?".

Reply:
Agreed, will update.